components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticator.java

```diff
@@ -911,13 +925,22 @@ public List<Property> getConfigurationProperties() {
         userIdLocation.setDisplayOrder(7);
         configProperties.add(userIdLocation);
 
+        Property scopes = new Property();
+        scopes.setName("oidcScopes");
```

Comment:
shall we introduce a constant for this `oidcScopes`. Refer FB https://github.com/wso2-extensions/identity-outbound-auth-facebook/blob/master/components/org.wso2.carbon.identity.application.authenticator.facebook/src/main/java/org/wso2/carbon/identity/application/authenticator/facebook/FacebookAuthenticator.java#L626